### DISCUSSION TITLE
[Nuxt 3] How to refresh async data on route change?

### DISCUSSION THREAD

**Discussion by samuelbednarcik** [nuxt/nuxt#17005] [Questions]
**Scenario**:
1. user comes to **auctions** page where auctions are fetched from API ( on server ) using **useAsyncData** composable ( **with only handler specified** )
2. user navigates to **homepage**
3. then, user navigates back to **auctions** page and data from the first visit is showed

I always want to show user the newest data ( data should be refetched when I come to the auctions page from a different page ). I can't figure out how to achieve this without duplicate fetching. I think it's a common use case for the majority of the applications.

**Comment by ghost** [comment]
```ts
await useAsyncData('init', () => init(), {
  initialCache: false
})
```
